fendermint/vm/message/src/ipc.rs

```diff
@@ -93,9 +94,19 @@ pub struct BottomUpCheckpoint {
     pub bottom_up_messages: Cid, // TODO: Use TCid
 }
 
+/// A proposal of the parent view that validators will be voting on.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub struct ParentFinalityProposal {
```

Comment:
```suggestion
pub struct ParentFinality {
```

I think we don't need to emphasize here that it's a proposal, after all everything in the block is a proposal at some point. What we want to express is that we view some height and hash as final, and this already says it in a way.